.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
These are still showing up. You can remove them from being tracked by git using: `git rm -r --cached .idea/`